### PR TITLE
IDE0008 Use explicit type instead of var

### PR DESCRIPTION
**Pull request by gerhardol**
Follow up to #11276 

## Proposed changes

Several manual changes, type alias, indentations, casing
A few IDE0090 was addressed too (that appeared after applying IDE0008

A few occurrences with anonymous types remains

I have not reviewed the changes (other than my manual overrides), it compiles and started up.

## Test methodology 

Compile

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by RussKie — APPROVED** [review]
+0

**Comment by vbjay** [comment]
I would then update editor config file to be at least a warning if not an error if you are taking this stance fully.

**Review by mstv — APPROVED** [review]
👍

File: UnitTests/GitCommands.Tests/AsyncLoaderTests.cs [hunk 1]
```diff
@@ -248,11 +248,11 @@ public void Error_raised_via_event_and_not_marked_as_handled_faults_task()
 
             Exception ex = new();
 
-            var loadTask = ThreadHelper.JoinableTaskFactory.RunAsync(() => _loader.LoadAsync(
+            Microsoft.VisualStudio.Threading.JoinableTask loadTask = ThreadHelper.JoinableTaskFactory.RunAsync(() => _loader.LoadAsync(
```
[the remark this answers is not in the document]
Comment (gerhardol) [reply UnitTests/GitCommands.Tests/AsyncLoaderTests.cs]: Also did this change in a general way, fixing IDE0001

**Comment by gerhardol** [comment]
Would like to merge this tomorrow to avoid merge conflicts - most other prs are affected by #11276 anyway.

**Comment by RussKie** [comment]
Merge at will

**Pull request merged**